seacrowd/sea_datasets/m3ls/m3ls.py

```diff
@@ -0,0 +1,392 @@
+"""
+SEA Crowd Data Loader for M3LS.
+"""
+import json
+import os
+from collections.abc import Iterable
+from copy import deepcopy
+from typing import Dict, Generator, List, Tuple, Union
+
+import datasets
+from datasets.download.download_manager import DownloadManager
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import TASK_TO_SCHEMA, Licenses, Tasks
+
+_CITATION = r"""
+@inproceedings{verma-etal-2023-large,
+    title = "Large Scale Multi-Lingual Multi-Modal Summarization Dataset",
+    author = "Verma, Yash  and
+      Jangra, Anubhav  and
+      Verma, Raghvendra  and
+      Saha, Sriparna",
+    editor = "Vlachos, Andreas  and
+      Augenstein, Isabelle",
+    booktitle = "Proceedings of the 17th Conference of the European Chapter of the Association for Computational Linguistics",
+    month = may,
+    year = "2023",
+    address = "Dubrovnik, Croatia",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/2023.eacl-main.263",
+    doi = "10.18653/v1/2023.eacl-main.263",
+    pages = "3620--3632",
+}
+"""
+
+logger = datasets.logging.get_logger(__name__)
+
+_LOCAL = False
+_LANGUAGES = ["ind"]
+
+
+_DATASETNAME = "m3ls"
+_DESCRIPTION = r"""
+The multilingual multimodal summarization dataset (M3LS) consists of over a million instances of document-image pairs
+along with a professionally annotated multimodal summary for each pair.
+It is derived from news articles published by the British Broadcasting Corporation (BBC) over a decade and spans 20 total languages,
+which Indonesian is the only SEA language available on this dataset.
+"""
+
+_HOMEPAGE = "https://github.com/anubhav-jangra/M3LS/tree/main"
+_LICENSE = Licenses.MIT.value
+
+_URL = "https://drive.google.com/uc?id=1Kznkw7YpRiWpdgH4_SVNwp0uGf3j-5e2"
+
+_SUPPORTED_TASKS = [Tasks.SUMMARIZATION, Tasks.IMAGE_CAPTIONING]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+_CONFIG_SUFFIXES_FOR_TASK = [TASK_TO_SCHEMA.get(task).lower() for task in _SUPPORTED_TASKS]
+
+
+class M3LS(datasets.GeneratorBasedBuilder):
+    """M3LS dataset of Indonesian Language (from BBC Indonesian)"""
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=datasets.Version(_SOURCE_VERSION),
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=_DATASETNAME)
+    ] + [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_{cfg_sufix}",
+            version=datasets.Version(_SEACROWD_VERSION),
+            description=f"{_DATASETNAME} seacrowd schema for {task.name}",
+            schema=f"seacrowd_{cfg_sufix}",
+            subset_id=_DATASETNAME)
+        for task, cfg_sufix in zip(_SUPPORTED_TASKS, _CONFIG_SUFFIXES_FOR_TASK)
+    ]
```

Comment:
This part does not pass the `black` formatter.
Use the following.

```suggestion
    BUILDER_CONFIGS = [
        SEACrowdConfig(
            name=f"{_DATASETNAME}_source",
            version=datasets.Version(_SOURCE_VERSION),
            description=f"{_DATASETNAME} source schema",
            schema="source",
            subset_id=f"{_DATASETNAME}",
        ),
        *[
            SEACrowdConfig(
                name=f"{_DATASETNAME}_seacrowd_{cfg_sufix}",
                version=datasets.Version(_SEACROWD_VERSION),
                description=f"{_DATASETNAME} seacrowd schema for {task.name}",
                schema=f"seacrowd_{cfg_sufix}",
                subset_id=f"{_DATASETNAME}",
            )
            for task, cfg_sufix in zip(_SUPPORTED_TASKS, _CONFIG_SUFFIXES_FOR_TASK)
        ],
    ]
```